source/includes/_payouts.md

```diff
@@ -50,12 +46,24 @@ Sign up for a developer account and get your personal API token for our sandbox.
 
 NB! Two factor authentication (2FA) code for sandbox login is 111111.
 
-Your developer account will have some test money that you can use to start making payments in same way as you would in a live environment. You get your API token in the Settings tab of your account page.
+Your developer account will have some test money that you can use to start making payments in same way as you would in a live environment. 
 
 Add your API token as header parameter to every request like this:
 
 *Authorization: Bearer xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx*
 
+### Acquiring your API token
+You get your API token in the Settings tab of your account page.
+
+### Limiting token access by IP
+You can optionally specify a number of IP addresses or IP ranges which calls to the API will be limited to. 
```

Comment:
Optionally you can enhance your integration security by restricting the calling IPs.

Each token access can be limited to single IPs or IP ranges in the  personal token view.

Please note that the ip whitelisting requires:
* single IP addresses to be only IPv4  e.g. `192.168.100.14`
* IP ranges should use [CIDR notation] ...

In case a request is being made using an IP that is not included in the whitelisted IPs, the server will respond with a 401 HTTP status code.